src/image-tags.js

```diff
@@ -1,56 +1,70 @@
-import {get, isEmpty} from 'lodash';
-import {FocusedImage} from 'quintype-js';
+import { get, isEmpty } from "lodash";
+import { FocusedImage } from "quintype-js";
 
 function pickImageFromCard(story, cardId) {
-  const { metadata = {} } = story.cards.find(card => card.id === cardId) || {};
-  if(metadata && !isEmpty(metadata) && get(metadata, ['social-share', 'image', 'key'], false)) {
-    return new FocusedImage(metadata['social-share'].image.key, metadata['social-share'].image.metadata || {});
+  const { metadata = {} } = story.cards.find((card) => card.id === cardId) || {};
+  if (metadata && !isEmpty(metadata) && get(metadata, ["social-share", "image", "key"], false)) {
+    const alt = metadata["social-share"].image.attribution || getAltribution(story);
+    return {
+      image: new FocusedImage(metadata["social-share"].image.key, metadata["social-share"].image.metadata || {}),
+      alt,
+    };
   }
 }
 
+function getAltribution(story) {
+  return (
+    story["hero-image-attribution"] ||
+    story.summary ||
+    get(story, ["alternative", "home", "default", "headline"]) ||
```

Comment:
for card sharing, first priority is for social-share title and message, if it is not present fallback to story headline
![image](https://user-images.githubusercontent.com/37729206/126752910-1794f2a4-50a0-4b20-a975-71c559541eb9.png)